<h2>Lab Alumni</h2>
<p><span style="text-decoration: underline;"><em>Postdocs</em></span></p>
<p>Bian Li, 2019 – 2021<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2019/08/Bian_Li-e1565329579678.jpg"><img loading="lazy" class="alignleft wp-image-579 " src="http://www.capralab.org/wordpress/wp-content/uploads/2019/08/Bian_Li-e1565329579678-234x300.jpg" alt="Bian Li" width="175" height="224" srcset="http://www.capralab.org/wordpress/wp-content/uploads/2019/08/Bian_Li-e1565329579678-234x300.jpg 234w, http://www.capralab.org/wordpress/wp-content/uploads/2019/08/Bian_Li-e1565329579678.jpg 429w" sizes="(max-width: 175px) 100vw, 175px" /></a>Bian received his PhD in Chemistry from Vanderbilt University in 2018. Prior to joining the Capra Lab, he was a postdoctoral associate at Yale University. Bian is primarily interested in computational structural biology and human molecular genetics. His research goals include understanding the genetic variants underlying differences in the efficacy and toxicity of drugs and developing data-driven tools for improving drug prescription. Bian&#8217;s previous research topics include drug discovery, protein structure prediction, and variant interpretation.</p>
<p> </p>
<p>David Rinker, 2016 – 2021<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2013/11/david-rinker.jpg" rel="attachment wp-att-265"><img loading="lazy" class="alignleft wp-image-265" src="http://www.capralab.org/wordpress/wp-content/uploads/2013/11/david-rinker.jpg" alt="david-rinker" width="170" height="211" /></a><br />David received his PhD in Genetics from Vanderbilt in 2015. His interests center around functional genomics, with a particular interest in exploring the relationship between gene regulation and phenotype. David’s previous research spanned both arthropod and fungal genomics, and he remains interested in placing genomic inquiries within the larger framework of comparative evolutionary biology.</p>
<p> </p>
<hr />
<p>Greg Sliwoski, July 2016 – May 2019<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2016/07/grs-e1470028530498.jpg" rel="attachment wp-att-265"><img loading="lazy" class="alignleft wp-image-297 size-full" src="http://www.capralab.org/wordpress/wp-content/uploads/2016/07/grs-e1470028530498.jpg" width="170" height="233" /></a>Greg received his PhD in Biochemistry from the University of Leipzig in Germany. He is primarily interested in exploring computational techniques that incorporate protein structure prediction into variant effect prediction. His research goals include improving rare variant analysis with structural modeling and applying these methods to personalized medicine and drug discovery. His previous research topics include computer-aided drug discovery, GPCR structure prediction and ligand docking, and QSAR descriptor development. Other interests include neuroscience and music. <em>Current Position:</em> Senior Data Scientist, <a href="https://www.loseit.com/">Lose It!</a>, Cambridge, MA</p>
<p> </p>
<h2>PhD Students</h2>
<p>Abin Abraham (MSTP), Fall 2017 – Present<br /><a class="alignleft" href="http://www.capralab.org/wordpress/wp-content/uploads/2017/08/abin_headshot-e1503757645448.jpg"><img loading="lazy" class=" wp-image-387" src="http://www.capralab.org/wordpress/wp-content/uploads/2017/08/abin_headshot-e1503757645448.jpg" alt="" width="146" height="199" /></a><br />Abin is a MD/PhD student in the Human Genetics program. He graduated from the University of Michigan with a B.S.E in Biomedical Engineering.</p>
<p> </p>
<p> </p>
<p>Evonne McArthur (MSTP/Human Genetics), Fall 2018 – Present<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2018/10/emcarthur_small-e1539585573397.jpg"><img loading="lazy" class="alignleft  wp-image-458" src="http://www.capralab.org/wordpress/wp-content/uploads/2018/10/emcarthur_small-e1539585573397.jpg" alt="" width="160" height="197" /></a><br />Evonne is a MD/PhD student in the Human Genetics program. She is interested in the role of structural and regulatory element variation in risk for both rare and common diseases. She graduated from the University of North Carolina at Chapel Hill with a B.S. in Quantitative Biology.</p>
<p> </p>
p>Mary Lauren Benton (DBMI), Summer 2016 – Summer 2020<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2016/05/Benton_pic-e1463893979316.jpg"><img loading="lazy" class="alignleft wp-image-287 size-full" src="http://www.capralab.org/wordpress/wp-content/uploads/2016/05/Benton_pic-e1463893979316.jpg" width="150" height="200" /></a><br />Mary Lauren was a graduate student in the BIDS program in the Department of Biomedical Informatics. She developed computational methods for comparing, predicting, and interpreting gene regulatory regions across tissues. She graduated with a B.S. in Bioinformatics from Baylor University.</p>
<p><em>Current Position</em>: Assistant Professor of Bioinformatics at <a href="https://www.ecs.baylor.edu/index.php?id=960856">Baylor</a></p>
<p> </p>
<p>Laura Colbran (Human Genetics), Summer 2016 – Summer 2020<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2016/05/DSC_0031-e1463894306493.jpg"><img loading="lazy" class="wp-image-286  alignleft" src="http://www.capralab.org/wordpress/wp-content/uploads/2016/05/DSC_0031-e1463894306493.jpg" width="145" height="194" /></a><br />Laura was a graduate student in the Human Genetics program. She applied machine learning methods to interpret modern and archaic gene regulatory sequences. She graduated with a B.S. in Biology from Carleton College.</p>
<p><em>Current Position:</em> Postdoc in the <a href="https://www.med.upenn.edu/mathieson-lab/">Mathieson Lab</a> at UPenn.</p>
<p> </p>
<p>Ling Chen (Biological Sciences), Summer 2014 – Winter 2019<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2015/05/ling.jpg"><img loading="lazy" class="alignleft wp-image-219" src="http://www.capralab.org/wordpress/wp-content/uploads/2015/05/ling.jpg" alt="ling" width="148" height="179" srcset="http://www.capralab.org/wordpress/wp-content/uploads/2015/05/ling.jpg 325w, http://www.capralab.org/wordpress/wp-content/uploads/2015/05/ling-247x300.jpg 247w" sizes="(max-width: 148px) 100vw, 148px" /></a><br />Ling was a graduate student in the Biological Sciences program. She developed machine learning methods to predict and interpret gene regulatory regions across species. She graduated with a B.S. in Biology from Nankai University.</p>
<p>Current position: Senior Data Scientist at <a href="https://www.illumina.com/">Illumina</a></p>
<p> </p>
<p>Alexandra Fish (Human Genetics), Summer 2012 – Spring 2017<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2014/08/Alex_Fish_small.jpg"><img loading="lazy" class="alignleft wp-image-105" src="http://www.capralab.org/wordpress/wp-content/uploads/2014/08/Alex_Fish_small.jpg" alt="Alex_Fish_small" width="148" height="168" /></a><br />Alex’s PhD research focused on identifying  genetic variants that regulate gene expression levels, with an emphasis on both long-range regulation and interactions between variants.  She has a B.S. in Biology and a B.S. in Psychology from UNC-Chapel Hill.<br /><em>Current Position: </em>Senior Scientist, <a href="http://www.genomicsplc.com" target="_blank" rel="noopener noreferrer">Genomics PLC</a>, Oxford, UK</p>
<p> </p>
<p>Corinne Simonti (Human Genetics), Summer 2013 – Spring 2017<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2014/01/corinne_photo2_small2.jpg"><img loading="lazy" class="alignleft wp-image-73" src="http://www.capralab.org/wordpress/wp-content/uploads/2014/01/corinne_photo2_small2-215x300.jpg" alt="corinne_photo2_small2" width="142" height="198" srcset="http://www.capralab.org/wordpress/wp-content/uploads/2014/01/corinne_photo2_small2-215x300.jpg 215w, http://www.capralab.org/wordpress/wp-content/uploads/2014/01/corinne_photo2_small2.jpg 720w" sizes="(max-width: 142px) 100vw, 142px" /></a><br />Corinne Simonti completed her PhD in the Human Genetics program at Vanderbilt University in 2017. Her primary research focused on determining the genetic basis of the phenotypic differences between humans and archaic hominins. She graduated with a B.S. in Genetics from the University of Georgia in 2012.<br /><em>Current Position: </em> Postdoc, <a href="https://popgen.gatech.edu" target="_blank" rel="noopener noreferrer">Lachance Lab</a>, Georgia Tech</p>
<p> </p>
<p>Mike Sivley (Biomedical Informatics), Summer 2013 – Winter 2017<br /><a href="http://www.capralab.org/wordpress/wp-content/uploads/2014/01/mike_profile_pic.jpg"><img loading="lazy" class="wp-image-69 alignleft" src="http://www.capralab.org/wordpress/wp-content/uploads/2014/01/mike_profile_pic-230x300.jpg" alt="MIKE!" width="143" height="185" /></a><br />Mike holds a BS and MS in Computer Science. His masters thesis focused on clustering rare genetic variation to improve statistical power for association studies. His PhD research developed analysis strategies for genetic variation in the context of protein structures and biobank data.<br /><em>Current Position: </em> Scientist, <a href="http://www.genomicsplc.com" target="_blank" rel="noopener noreferrer">Genomics PLC</a>, Oxford, UK</p>
<p> </p>
<p><span style="text-decoration: underline;"><em>Vanderbilt Undergrads</em></span></p>
<p>Elizabeth Dorans, Undergrad Intern, 2019–2020</p>
<p>Kevin Yang, Undergrad Intern, Summer 2018 – Summer 2019</p>
<p>Daniel Yan, Undergrad Intern, Summer 2018</p>
<p>Xiaoyi Dou, Undergrad Intern, Summer 2016 – Spring 2017, Summer 2018</p>
<p>Chantay Young, Undergrad Intern, Summer 2017<br /><!--
<p> </p>

